performance/README.md

```diff
@@ -0,0 +1,59 @@
+## Setup
+To run locally first install k6. The browser module is now included in the default install(https://github.com/grafana/k6)
+`brew install k6`
+
+To run the tests head/headless you will need chromium installed - to install chromium run
+`brew install --force chromium --no-quarantine ` 
+
+## Test Configuration
+Tests are designed to allow running browser and API tests independently or in parallel. You can monitor the UI while applying different API load scenarios.
+
+Set the SCENARIO environment variable to control what test(s) are executed.
+
+Available Scenarios:
+
+BROWSER – Browser-based UI test
+
+API – Performance/load test scenarios:
+
+- load
+- soak
+- smoke
+- stress
+
+## Running the tests
+To run the tests locally these can be executed from the command line in a terminal. 
+From the root of the project, you can run the following commands, or navigate to the directory of the run_test.sh file and run that (you can configure which scenario to run in there too).
+
+- Browser test only
+`SCENARIO=browser k6 run performance/tests/browserAndApi.js`
+
+- API tests
+Before running those, you need to capture the one-time handover link as  extracted url. The extract scenario must be run first to get that value, which is run as part of the execution of the bash script.
+
+## Technical description:
+
+The run_test.sh script is a Bash automation wrapper designed to run a two-phase performance testing workflow using k6. It first executes a browser-based extraction step to retrieve a dynamic URL, then passes that URL as an environment variable to run following API-based performance tests.
+
+To use the run_test.sh file to run the api tests, navigate to the `utils` directory and run:
+```chmod +x run_test.sh
```

Comment:
you need to break the three backticks onto their own lines - check the preview at https://github.com/ministryofjustice/hmpps-arns-integrations-test-playwright-poc/blob/ACE-95-implement-performance-tests/performance/README.md

Reply:
Ah sorry. Okay!